release/models/gnsi/openconfig-gnsi-certz.yang

```diff
@@ -0,0 +1,204 @@
+module openconfig-gnsi-certz {
+  yang-version 1;
+  namespace "https://github.com/openconfig/yang/gnsi/certz";
+  prefix oc-gnsi-certz;
+
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-system {
+    prefix oc-sys;
+  }
+  import openconfig-system-grpc {
+    prefix oc-sys-grpc;
+  }
+  import openconfig-types {
+    prefix oc-types;
+  }
+  import openconfig-yang-types {
+    prefix oc-yang;
+  }
+  import openconfig-gnsi {
+    prefix oc-gnsi;
+  }
+  organization
+    "OpenConfig Working Group";
+
+  contact
+    "OpenConfig working group
+    netopenconfig@googlegroups.com";
+
+  description
+    "This module provides a data model for the metadata of gRPC credentials
+    installed on a networking device.";
+
+  oc-ext:openconfig-version "0.5.0";
+
+  revision 2024-01-25 {
+    description
+      "Major style updates.";
+    reference "0.5.0";
+  }
+
+  revision 2023-08-24 {
+    description
+      "Adds ssl-profile-id leaf";
+    reference "0.4.0";
+  }
+
+  revision 2023-05-10 {
+    description
+      "Adds authentication policy freshness information.";
+    reference "0.3.0";
+  }
+
+  revision 2022-10-30 {
+    description
+      "Adds success/failure counters.";
+    reference "0.2.0";
+  }
+
+  revision 2022-09-20 {
+    description
+      "Initial revision.";
+    reference "0.1.0";
+  }
+
+  typedef version {
+    type string;
+    description
+      "The version ID of the credential as provided by the credential
+      manager when the credential was pushed. This leaf persists through
+      a reboot.";
+  }
+
+  typedef created-on {
+    type oc-types:timeticks64;
+    description
+      "The creation time of the credential as reported by the credential
+      manager when the credential was pushed to the device. This value is
+      reported as nanoseconds since epoch (January 1st, 1970 00:00:00 GMT).
+      This leaf persists through a reboot.";
+  }
+  // gRPC server related definitions.
+  // Success/failure counters.
+  grouping grpc-server-acctz-counters  {
+    description
+      "A collection of counters that were collected while evaluating
+      access to the gRPC server.";
+
+    container certz-counters {
```

Comment:
Could you give more details on these counters and why they should be in certz?

I am assuming that:
- access-accepts increments when there is a successful TLS handshake
- access-rejects increments when there is a TLS handshake failure

Is my understanding correct?

Reply:
That sounds correct to me.   Wen, I suggest:

"The total number of times a TLS handshake failure has occurred and the gRPC server denied access a client.";


"The total number of times a successful TLS handshake is completed and the gPRC server allows access to a client.";

Reply:
Thanks, Done.